modules/msk/custom/splitfastqbylane/tests/tags.yml

```diff
@@ -0,0 +1,2 @@
+custom/splitfastqbylane:
```

Comment:
Would you maybe want to add a `nextflow.config` to the `tests` directory? I don't think it's necessary, but might be nice to have the template for [specifying process parameters and the publish directory](https://github.com/mskcc-omics-workflows/modules/blob/72054cdc64de93b770040738bd3a2839cf5bb907/modules/msk/gbcms/tests/nextflow.config).

Reply:
i think that might be a good general practice, but i wasn't sure about this for this particular module. I don't think this type of process will typically be published, maybe only used as an intermediate step? also although i define args in the script block, it's never actually used in the shell execution, so i am thinking that i should probably remove them. given that info do you think it's still a good idea to add the file?